dev/tools/gen_defaults/data/shape.json

```diff
@@ -0,0 +1,94 @@
+{
+  "version": "v0_76",
+
+  
+    "md.sys.shape.corner.extra-large": {
+      "family": "SHAPE_FAMILY_ROUNDED_CORNERS",
+      "topLeft": 28.0,
+      "topRight": 28.0,
+      "bottomLeft": 28.0,
+      "bottomRight": 28.0
+    },
+  
+  
+    "md.sys.shape.corner.extra-large.top": {
+      "family": "SHAPE_FAMILY_ROUNDED_CORNERS",
+      "topLeft": 28.0,
+      "topRight": 28.0,
+      "bottomLeft": 0.0,
+      "bottomRight": 0.0
+    },
+  
+  
+    "md.sys.shape.corner.extra-small": {
+      "family": "SHAPE_FAMILY_ROUNDED_CORNERS",
+      "topLeft": 4.0,
+      "topRight": 4.0,
+      "bottomLeft": 4.0,
+      "bottomRight": 4.0
+    },
+  
+  
+    "md.sys.shape.corner.extra-small.top": {
+      "family": "SHAPE_FAMILY_ROUNDED_CORNERS",
+      "topLeft": 4.0,
+      "topRight": 4.0,
+      "bottomLeft": 0.0,
+      "bottomRight": 0.0
+    },
+  
+  
```

Comment:
Extra new lines between each entry in this file

Reply:
Fixed by having the extractor script run clang format over the files.